src/style.css

```diff
@@ -1,3 +1,61 @@
+:root {
+  --main-blue-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 40px;
+}
+
+.navigation {
+  margin: 0;
+}
+
+.nav-link-list {
+  display: flex;
+  margin: 0;
+}
+
+.link-item {
+  list-style: none;
+  line-height: 60px;
+  position: relative;
+}
+
+.link-item:not(:last-child) {
+  margin-right: 20px;
+  margin-left: 20px;
+}
+
+.link-item a {
+  text-decoration: none;
+  color: #000;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
+.link-item a:hover,
+.link-item a.is-active,
+.link-item a:active{
+  color: var(--main-blue-color);
+}
+
+.link-item a:hover::after,
+.link-item a:active::after {
```

Comment:
The `::after` pseudo-element should be used for the blue line below the active link, not just on hover. This is crucial for indicating the current active page to users. Please ensure that the active link styling includes the `::after` pseudo-element.